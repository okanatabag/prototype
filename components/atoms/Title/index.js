<<<<<<< HEAD
import styled from "styled-components";

const Title = styled.h2`
	margin: ${(props) => props.mrg || "0"};
	font-size: ${(props) => (props.size ? `${props.size * 0.8}rem` : "2rem")};
	color: ${(props) => props.color || "#000"};
	font-weight: 600;
	@media only screen and (max-width: 520px) {
		font-size: ${(props) =>
			props.size ? `${props.size * 0.8}rem` : "1.6rem"};
	}
	@media only screen and (max-width: 460px) {
		font-size: ${(props) =>
			props.size ? `${props.size * 0.6}rem` : "1.2rem"};
	}
	@media only screen and (max-width: 360px) {
		font-size: ${(props) =>
			props.size ? `${props.size * 0.5}rem` : "1rem"};
	}
`;

export default Title;
=======
import styled from "styled-components"

const Title = styled.h2`
    margin: ${props => props.mrg || "0"};
    font-size:${props => props.size ? (props.size*0.8)+'rem' : "2rem"};
    color: ${props => props.color || "#000"};
    font-weight: 600;
    @media only screen and (max-width: 520px) {
        font-size: ${props => props.size ? (props.size*0.8)+'rem' : "1.6rem"};
    }
    @media only screen and (max-width: 460px) {
        font-size: ${props => props.size ? (props.size*0.6)+'rem' : "1.2rem"};
    }
    @media only screen and (max-width: 360px) {
        font-size: ${props => props.size ? (props.size*0.5)+'rem' : "1rem"};
    }
`;

export default Title;
>>>>>>> b9cde6f17d12c509c1e9bb7145254a0b4fa476c1
