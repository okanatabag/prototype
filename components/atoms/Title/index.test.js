<<<<<<< HEAD
import React from "react";
import renderer from "react-test-renderer";
import Title from ".";

it("Title renders correctly", () => {
	const tree = renderer
		.create(
			<Title size="5" mrg="5px" color="#ff0000">
				Test
			</Title>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
=======
import React from 'react';
import renderer from 'react-test-renderer';
import Title from '.'
it('Title renders correctly', () => {
    const tree = renderer
      .create(<Title size="5" mrg="5px" color="#ff0000">Test</Title>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});
>>>>>>> b9cde6f17d12c509c1e9bb7145254a0b4fa476c1
