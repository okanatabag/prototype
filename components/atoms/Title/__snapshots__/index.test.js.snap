// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<h2
  className="Title-sc-156iwuq-0 bLOHaJ"
  color="#ff0000"
  size="5"
>
  Test
</h2>
`;
